### PR TITLE
Improve the Fedora package-buidlers for better debugability

### PR DESCRIPTION
**Pull request by prologic**

### CODE REVIEW

**Comment by Ferroin** [comment]
@prologic Could you just extend this to cover all the package builders? Seems kind of pointless to do it only for one distro.

We also need a way to make sure this only triggers in interactive usage of the containers, and not during automated builds, otherwise we will end up causing failed package builds to hang in CI.

File: package-builders/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+fail() {
+  printf "FAIL: %s\n" "$1"
+  if [ -t 1 ]; then
```
Comment (prologic) [inline package-builders/entrypoint.sh:5]: @Ferroin The "interactive check" is here.

**Comment by prologic** [comment]
> @prologic Could you just extend this to cover all the package builders? Seems kind of pointless to do it only for one distro.
> 
> We also need a way to make sure this only triggers in interactive usage of the containers, and not during automated builds, otherwise we will end up causing failed package builds to hang in CI.

Yes you are right. I will do this now. I was backed into a corner when doing this initially. I have some breathing room now :)

**Comment by prologic** [comment]
@Ferroin Done

**Comment by prologic** [comment]
@Ferroin Can I merge this? :)

**Comment by prologic** [comment]
@Ferroin if you can re-review this #76 will make more sense as it needs the same improvements at the bottom :)

**Pull request merged**